hybrasyl/Inventory.cs

```diff
@@ -500,7 +500,100 @@ public GuildVault(string ownerUuid) : base(ownerUuid)
         public GuildVault(string ownerUuid, uint goldLimit, ushort itemLimit) : base(ownerUuid, goldLimit, itemLimit) { }
     }
 
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ParcelStore
+    {
+        private readonly object _lock = new object();
+
+        [JsonProperty]
+        public string OwnerUuid { get; set; }
+        [JsonProperty]
+        public Dictionary<string, (string Sender, string Item, uint Quantity)> Items { get; set; } //storage id, named tuple

```

Comment:
I'm confused why this is a dictionary when you never use the keys externally (since retrieval is done by a User object reference)? 

Seems like it could just be `List<Parcel>`, parcel being a simple class with Sender/Item/Quantity/Gold?

I could be missing something here though.

Reply:
gold/item are two separate objects for sending/receiving. gold is a forward thinking item. I did it this way so that the key is a hashcode generated from the sending user and the time sent.  this could be a list of parcel, but I didn't want to have duplicate entries without a unique key.